python/doc/user_manual/functions.rst

```diff
@@ -83,6 +83,8 @@ Refer to :ref:`field_function`.
     OpenTURNSPythonFieldFunction
     OpenTURNSPythonFieldToPointFunction
     OpenTURNSPythonPointToFieldFunction
+
+    :template: classWithPlot.rst_t
     P1LagrangeInterpolation
```

Comment:
if you want to use classWithPlot template then you should provide the python/doc/pyplots/P1LagrangeInterpolation.py which is missing, maybe you forgot to "git add" it ?

Reply:
There was 2 errors here!

1. the P1LagrangeInterpolation does not exist : I removed the line from the function.rst
2. the P1LagrangeEvaluation was not associated to a classWithPlot: I introduced the line